websites/O/osu ppy/presence.ts

```diff
@@ -12,50 +61,65 @@ let beatmapTitle: string,
 	selected: Element,
 	gamemode: string,
 	forumName: string,
-	inputSelected: HTMLInputElement;
+	inputSelected: HTMLInputElement,
+	strings: Awaited<ReturnType<typeof getStrings>>,
+	oldLang: string = null;
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-		largeImageKey:
-			"https://cdn.rcd.gg/PreMiD/websites/O/osu%20ppy/assets/logo.png",
-	};
+			largeImageKey: "logo",
```

Comment:
Use an image URL over Discord asset name.

Reply:
> Use an image URL over Discord asset name.

I’ll resolve this tomorrow.